src/js/models/card-node.js

```diff
@@ -35,7 +35,8 @@ export default class CardNode {
         this.display();
       },
       cancel: () => this.display(),
-      remove: () => this.remove()
+      remove: () => this.remove(),
+      card: this
```

Comment:
https://github.com/bustlelabs/content-kit-editor/blob/master/CARDS.md should be updated as well.